Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -104,7 +104,7 @@ - (void)createViews {
 
     UILabel *titleLabel = [UILabel new];
     titleLabel.textColor = [UIColor whiteColor];
-    titleLabel.text = NSLocalizedString(@"VERIFICATION_HEADER", @"Navigation title in the registration flow - during the sms code verification process.");
+    titleLabel.text = [self phoneNumberText];
```

Comment:
I thought the previous header, "Verify" wasn't adding any value. 

If the number was entered incorrectly, I thought putting it near the back button would draw the eye nearer to the next step (going "back" and fixing it.)